Tasks/Common/artifacts-common/connectionDataUtils.ts

```diff
@@ -29,6 +29,35 @@ export async function getConnectionDataForProtocol(protocolType: protocols.Proto
     }, 3, 1000);
 }
 
+/**
+ * Gets the api location url for any feed given the location id.
+ * @param protocolType Packaging protoocol like "NuGet", "PyPI"
+ * @param apiVersion Api version of the endpoint. ex: 3.0-preview, 5.0
+ * @param locationGuid location id for an api. ex: 93377A2C-F5FB-48B9-A8DC-7781441CABF1 for PyPi simple api
+ * @param feedId FeedId to put in api params
+ * @param project projectId to put in api params
+ */
+export async function getPackagingEndpointUrl(
```

Comment:
Suggest rename to "getPackagingRouteUrl" or something like that.
Also, this function assumes that the only route parameters that need to be filled are feedId and project, so if a route has other inputs this will return a route template rather than a route I think? Should make that clear and possibly allow specifying additional route values.

Reply:
cool, I thought we won't need any other params in auth tasks. But, I can add an optional dictionary here.

Reply:
A short comment explaining the above would be fine instead of implementing it now.
e.g. this function wouldn't work for routes that have {packageName}, {packageVersion}, etc if we ever needed to get one of those.

Reply:
cool, will do. thanks!